compiler/x/codegen/X86Ops.hpp

```diff
@@ -1217,6 +1217,102 @@ typedef enum
    IA32EFlags_CF = 0x10
    } TR_EFlags;
 
+namespace TR
+   {
+   namespace X86
+      {
+      namespace Instruction
```

Comment:
This would be the first use of namespaces under `TR::` right? 

(Note: Just because it's different doesn't mean we shouldn't merge it, but it's certainly a bit curious compared to other uses of namespaces we have. )